plugins/kubernetes/src/index.ts

```diff
@@ -24,8 +24,12 @@ export {
   kubernetesPlugin,
   kubernetesPlugin as plugin,
   EntityKubernetesContent,
+  EntityKubernetesContent as Router,
 } from './plugin';
-export type { EntityKubernetesContentProps } from './plugin';
-export { Router, isKubernetesAvailable } from './Router';
-// TODO remove this re-export as a breaking change after a couple of releases
-export * from '@backstage/plugin-kubernetes-react';
```

Comment:
I noticed we removed the re-export. I think we should leave this in for a little longer